src/main/java/bowling/domain/Participant.java

```diff
@@ -0,0 +1,21 @@
+package bowling.domain;
+
+public class Participant {
+
+    private String participant;
```

Comment:
참가자의 속성중에 이름이 있어야 하니 속성 명은 `name`이 조금 더 명시적일 것 같습니다 🙏 
그리고 `final` 도 추가 되면 더 좋을 것 같습니다 😄 
```suggestion
    private final String name;
```

Reply:
네 참가자라는 객체를 만들었으니 `name` 이라는 필드가 명시적인거 같습니다!